src/udev/scsi_id/scsi_id.c

```diff
@@ -228,8 +232,10 @@ static void help(void) {
                "  -g --allowlisted                 Treat device as allowlisted\n"
                "  -u --replace-whitespace          Replace all whitespace by underscores\n"
                "  -v --verbose                     Verbose logging\n"
-               "  -x --export                      Print values as environment keys\n",
-               program_invocation_short_name);
+               "  -x --export                      Print values as environment keys\n"
+               "  -i --sgio                        Use sg_io for VPD pages\n"
+               "  -y --sysfs                       Use sysfs for VPD pages\n"
```

Comment:
`for VPD pages` -> `for reading VPD pages` ?

Reply:
Done